cfme/intelligence/reports/reports.py

```diff
@@ -539,7 +539,6 @@ def delete(self, cancel=False):
             view.flash.assert_no_error()
         else:
             view.flash.assert_no_error()
-            view.flash.assert_message("Successfully deleted Saved Report from the CFME Database")
```

Comment:
**REQUIRED**
Why it's deleted?

Reply:
Hey when we delete the saved report it doesn't display the flash message. Its bug.I logged that.This is reproducible in 5.7 and 5.8 version. so i think its not required. 
Just have a look ---> https://bugzilla.redhat.com/show_bug.cgi?id=1489387

Reply:
Don't delete this line, just comment it and add TODO about this bug

Reply:
Yes right. I will comment.